clients/java/signalr/src/main/java/com/microsoft/signalr/HttpHubConnectionBuilder.java

```diff
@@ -76,7 +75,7 @@ public HttpHubConnectionBuilder withAccessTokenProvider(Single<String> accessTok
      * @param timeout The duration that the {@link HubConnection} should wait for a Handshake Response from the server.
      * @return This instance of the HttpHubConnectionBuilder.
      */
-    public HttpHubConnectionBuilder withHandshakeResponseTimeout(Duration timeout) {
+    public HttpHubConnectionBuilder withHandshakeResponseTimeout(long timeout) {
```

Comment:
Make the *parameter* name `timeoutInMilliseconds`. I agree putting it in the method name would be a bit much.

Reply:
```suggestion
    public HttpHubConnectionBuilder withHandshakeResponseTimeout(long timeoutInMilliseconds) {
```